### DISCUSSION TITLE
Claude.ai (Anthropic) or OpenAI to create workflows

### DISCUSSION THREAD

**Discussion by siloteam** [clientIO/joint#2830] [Q&A]
### Introduction

Has anyone used JointJS with Claude.ai (Anthropic) or OpenAI to create workflows?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by marekhozak10** [comment]
Hi Rasmus, thanks for your post here! 👋 I'll take the liberty of following up on our conversation via email. 

There are two applications you might want to check out:

**1. Appmixer**
Our second product (an embedded integration platform) that uses JointJS+ and, more recently, AI components as part of workflows. You can get a good idea of what I mean by visiting the Appmixer website: https://www.appmixer.com/. If you'd like to explore further, we’d be happy to show you the platform in a 1:1 call.

**2. Our AI mind map generator demo app**
You may have seen this one already: https://www.jointjs.com/demos/chatgpt-mindmap. It gathers user input, sends a request to ChatGPT, performs some light post-processing on the received data, and renders it in the form of an editable mind map. This week, we're releasing a step-by-step guide on how we built this application (I'll share it here with you).

Do any of these help you get a better idea of how JointJS and AI can be combined to create workflows? :)

**Reply by marekhozak10** [reply]
@siloteam Here's the article I mentioned earlier: https://www.jointjs.com/blog/diagramming-meets-ai

**Comment by siloteam** [comment]
This is amazing! Thank you so much and I will sync with the team and see
how we can approach this in the best way! Very exciting! Talk soon mate!
